src/c_api/c_api.cc

```diff
@@ -823,9 +823,9 @@ int MXLoadLib(const char *path) {
 
       // MXNET_REGISTER_SUBGRAPH_PROPERTY(customBackend, CustomSubgraphProperty);
       mxnet::op::SubgraphBackendRegistry::Get()->__REGISTER_CUSTOM_PROPERTY__(name_str,
-                            std::make_shared<mxnet::op::CustomSubgraphProperty>(
-                           strategy_str, callSupportedOps, supportedOps_fp,
-                           callAcceptSubgraph, acceptSubgraph_fp, callFree, op_name_str));
+                               std::make_shared<mxnet::op::CustomSubgraphProperty>(
```

Comment:
looks like some unintentional spacing changes were made. can you please fix them if you end up pushing another commit?

Reply:
sure, will do